lab-davidw/test/homes-test.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
```

Comment:
Nice tests so far. Going forward with testing, I would try to test whether specific values are the same. Ex. Input = output